examples/tree_1d_dgsem/elixir_euler_quasi_1d_discontinuous.jl

```diff
@@ -0,0 +1,85 @@
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# Semidiscretization of the quasi 1d compressible Euler equations
+# See Chan et al.  https://doi.org/10.48550/arXiv.2307.12089 for details
+
+equations = CompressibleEulerEquationsQuasi1D(1.4)
+
+"""
+    initial_condition_discontinuity(x, t, equations::CompressibleEulerEquations1D)
+
+An entropy conservation verification initial condition taken from
```

Comment:
```suggestion
A discontinuous initial condition taken from
```
Since we don't use this IC for entropy conservation verification, I suggest we remove that from the description so it's not misleading.